gateway-ha/src/main/java/io/trino/gateway/ha/config/RulesExternalConfiguration.java

```diff
@@ -39,4 +40,14 @@ public void setExcludeHeaders(List<String> excludeHeaders)
     {
         this.excludeHeaders = excludeHeaders;
     }
+
+    public boolean isPropagateErrors()
+    {
+        return this.propagateErrors;
+    }
+
+    public void setPropagateErrors(Boolean propagateErrors)
```

Comment:
is `Boolean` (object wrapper) intened?
```suggestion
    public void setPropagateErrors(boolean propagateErrors)
```